show/main.py

```diff
@@ -2424,5 +2424,8 @@ def tunnel():
 
     click.echo(tabulate(table, header))
 
+from .kube import kubernetes
```

Comment:
Can you move this import to the top of the file now? Are the cyclical dependencies fixed?

Reply:
Yes & moved.